hack/vmanomaly-lint-hack.yaml

```diff
@@ -2,8 +2,17 @@ eula: true
 remote:
   read:
     url: http://cluster-victoria-metrics-cluster-vmselect.default.svc.cluster.local:8481/select/
+    verify_tls: true
+    basicAuth:
+      username: asdf
+      password: asdf
+    extra_filters:
+      - '{cluster!=""}'
+      - '{job!=""}'
   write:
     url: http://cluster-victoria-metrics-cluster-vminsert.default.svc.cluster.local:8480/insert/
+    verify_tls: false
+    bearer_token: asdf
```

Comment:
should we support file, secret?

Reply:
good point, secret should be an option

Reply:
hm, but atm there is no way to pass either an env variable or a path to the token to vmanomaly
wdyt about adding this in a next version? (after adding this to vmanomaly first)

Reply:
sure, it works for me